apps/api/src/app/workflows-v2/exceptions/step-has-issues.exception.ts

```diff
@@ -0,0 +1,8 @@
+import { InternalServerErrorException } from '@nestjs/common';
+import { NotificationStepEntity } from '@novu/dal';
+
+export class StepHasIssuesException extends InternalServerErrorException {
+  constructor(currentStep: NotificationStepEntity) {
+    super({ message: 'Step has internal issues, missing id or name', step: currentStep });
```

Comment:
This error message is confusing. Is it about a missing id or name or is it about the issues each step can have for its controls?